src/main/java/frc/robot/Constants.java

```diff
@@ -38,10 +38,11 @@
 
     public static final double DISTANCE_PER_PULSE = 0.0001; // TODO: find real number ~ This is the coefficient of encoder ticks to convert to distance
     
-
     //SPARKMAX for the spinner (Carousel)
     public static final int SPINNER_CAN_ID = 6; // CAN ID for the spinner for carousel
 
+    public static final double CAROUSEL_SPEED = 0.5; // This is just percent output
```

Comment:
Actually, we're going to need two carousel speeds, a slow speed while we're intaking balls and a high speed for shooting.